src/pkg/cluster/zarf.go

```diff
@@ -187,6 +187,16 @@ func (c *Cluster) RecordPackageDeployment(ctx context.Context, pkg v1alpha1.Zarf
 		componentWebhooks = existingPackageSecret.ComponentWebhooks
 	}
 
+	// TODO: This is done for backwards compartibility and could be removed in the future.
+	connectStrings := types.ConnectStrings{}
```

Comment:
Do we want to keep this? I cant find any code that is reading this from the Zarf state.

Reply:
Yeah the one situation where I thought we might need it `zarf connect list` pulls the info direct from cluster. If I was to guess the reason it's on here was for the deprecated Zarf UI, though that may be incorrect. IMO we should remove it from deployed package.

Reply:
Lets leave it and come back at a later time and remove it in its own PR.